test/aws/default/verify/controls/aws_route_tables.rb

```diff
@@ -0,0 +1,28 @@
+fixtures = {}
+[
+  'routetable_rtb_route_table_id',
+  'routetable_rtb_route_table_2_id',
+  'routetable_rtb_vpc_id',
+  'routetable_rtb_route_table_2_id'
+].each do |fixture_name|
+  fixtures[fixture_name] = attribute(
+    fixture_name,
+    default: "default.#{fixture_name}",
+    description: 'See ../build/ec2.tf',
+  )
+end
+
+control "aws_route_tables recall" do
+  # You should be able to get a specific subnet given its id
+  describe aws_route_tables do
+    it { should exist }
```

Comment:
I'd suggest also adding a check that the results include one of the route tables you created:
`its('route_table_ids') { should include fixtures['route_table_1_id'] }`